composer.json

```diff
@@ -82,10 +81,7 @@
         ]
     },
     "config": {
-        "preferred-install": "dist",
-        "platform": {
-            "php": "7.2.5"
```

Comment:
This is to prevent differences between the marketplace delivered version and the composer one, but perhaps it's finally been fixed on the marketplace side by @daftspunk?